cmd/main.go

```diff
@@ -72,11 +88,23 @@ func main() {
 			return
 		}
 		//
+		// Delete report.
```

Comment:
Should we wait until we know whether or not windup succeeded to delete the old report?

Reply:
No. Ramon has asked for the previous report to always be deleted.